region/src/main/java/org/motechproject/nms/region/domain/State.java

```diff
@@ -35,6 +41,10 @@
     private List<District> districts;
 
     @Field
+    @Cascade(delete = true)
+    @Persistent(table = "nms_states_join_circles", mappedBy = "states", defaultFetchGroup = "true")
+    @Join(column = "state_id")
+    @Element(column = "circle_id")
     private List<Circle> circles;
```

Comment:
and the query in LanguageServiceImpl that mentions the table name.

On Tue, Jun 9, 2015 at 8:58 PM, Lauren Lavoie <notifications@github.com>
wrote:

> In region/src/main/java/org/motechproject/nms/region/domain/State.java
> <https://github.com/motech-implementations/mim/pull/333#discussion_r32086485>
> :
>
> > @@ -35,6 +41,10 @@
> >      private List<District> districts;
> >
> >      @Field
> > +    @Cascade(delete = true)
> > +    @Persistent(table = "nms_states_join_circles", mappedBy = "states", defaultFetchGroup = "true")
> > +    @Join(column = "state_id")
> > +    @Element(column = "circle_id")
> >      private List<Circle> circles;
>
> @larubbio <https://github.com/larubbio> Do we just need this change
> (lines 44-47) and the change in LanguageServiceImpl.java?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/333/files#r32086485>.
>